src/box/space.c

```diff
@@ -303,6 +304,7 @@ space_create(struct space *space, struct engine *engine,
 	}
 	space->constraint_ids = mh_strnptr_new();
 	rlist_create(&space->memtx_stories);
+	space->wal_ext = wal_ext_get_by_name(space_name(space));
```

Comment:
Bad name - we need to emphasize that this function returns a WAL extension configuration for a *space*.

Reply:
Renamed to `wal_ext_get_by_space_name`.